src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerfileFunctionalTest.groovy

```diff
@@ -3,6 +3,7 @@ package com.bmuschko.gradle.docker.tasks.image
 import com.bmuschko.gradle.docker.AbstractFunctionalTest
 import org.apache.commons.lang.exception.ExceptionUtils
 import org.gradle.api.tasks.TaskExecutionException
+import org.gradle.internal.impldep.com.google.common.io.Files
```

Comment:
Again, no need to use internal files. OOTB java functions exist for doing this very thing.